src/Chapter4.hs

```diff
@@ -472,10 +486,11 @@ Implement the Applicative instance for our 'Secret' data type from before.
 -}
 instance Applicative (Secret e) where
     pure :: a -> Secret e a
-    pure = error "pure Secret: Not implemented!"
+    pure a = Reward a
```

Comment:
If you remember eta-reduction from Chapter Two, then you can try to eta-reduce this definition as well :slightly_smiling_face: